agent/bench-scripts/pbench-specjbb2005

```diff
@@ -276,17 +276,16 @@ fi
 trap "interrupt" INT QUIT TERM
 
 # edit the jbb properties file to match user's options
-> ${benchmark_run_dir}/SPECjbb.props.sed
-printf -- "s/#?input.jvm_instances=.*/input.jvm_instances=$nr_jvms/\n" >> ${benchmark_run_dir}/SPECjbb.props.sed
-printf -- "s/#?input.measurement_seconds=.*/input.measurement_seconds=$runtime/\n" >> ${benchmark_run_dir}/SPECjbb.props.sed
-printf -- "s/#?input.starting_number_warehouses=.*/input.starting_number_warehouses=$start_warehouses/\n" >> ${benchmark_run_dir}/SPECjbb.props.sed
-printf -- "s/#?input.increment_number_warehouses=.*/input.increment_number_warehouses=$inc_warehouses/\n" >> ${benchmark_run_dir}/SPECjbb.props.sed
-printf -- "s/#?input.ending_number_warehouses=.*/input.ending_number_warehouses=$stop_warehouses/\n" >> ${benchmark_run_dir}/SPECjbb.props.sed
-sed -E -f ${benchmark_run_dir}/SPECjbb.props.sed ${specjbb2005_dir}/SPECjbb.props > ${benchmark_run_dir}/SPECjbb.props
+cp ${specjbb2005_dir}/SPECjbb.props ${benchmark_run_dir}/SPECjbb.props
 if [ $? -ne 0 ]; then
 	printf "\t${benchmark}: unable to create the SPECjbb.props file (\"${benchmark_run_dir}/SPECjbb.props\")\n\n"
 	exit 1
 fi
+printf "input.jvm_instances=$nr_jvms\n" >> ${benchmark_run_dir}/SPECjbb.props
```

Comment:
So I assume "we're confident" that leaving current (uncommented) lines in the default alone and overriding them with new lines at the end actually works correctly? Our test scripts use `_replace_` or `_bad_` and I guess I never really thought about what was in the real ones as the replacement was pretty generic.

Reply:
This change came from https://github.com/portante/pbench/pull/20.  I did not personally verify this change, so I don't have confidence in it.  But if somebody can provide the confidence, great.

Reply:
I personally verified this change, by installing SpecJBB2005 and running it with `pbench-specjbb2005`, using the default (and problematical) config file, with several sets of command line switches, and the config lines which the script appends to the config file produce the desired effects, including overriding each of commented-out and non-commented-out config parameters.